Duke.java

```diff
@@ -0,0 +1,216 @@
+import javax.management.InstanceNotFoundException;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+public class Duke {
+    private String name = "Lakinta";
+    private Scanner scanner = new Scanner(System.in);
+    private ArrayList<String> tasks = new ArrayList<>();
+    private ArrayList<Task> taskArrayList = new ArrayList<>();
+    private int count = 1;
```

Comment:
I think since this count variable is now only called by the setTasks method, you can reduce the scope to follow coding standards (by initializing it inside the method itself) 
![image](https://github.com/nus-cs2103-AY2324S1/ip/assets/99804772/ca15c966-c290-4cb8-9b7a-e8572fac7792)